src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/tasks/TasksSchedulingConfiguration.kt

```diff
@@ -26,8 +24,7 @@ class TasksSchedulingConfiguration(
   @Scheduled(cron = "\${CRON_IMPORT_REPORTS}")
   @SchedulerLock(name = "importReports")
   fun previousDaysReportsTask() {
-    previousDaysMoves.execute()
-    previousDaysPeopleAndProfiles.execute()
+    importReportsDataTask.execute()
```

Comment:
We now only have one task responsible for importing all data.  This is to make sure all files are present prior to importing.  When split into two the two tasks do not know about each other.